Jenkinsfile

```diff
@@ -87,6 +87,9 @@ timestamps {
 				currentBuild.description = currentBuild.description + " => failed"
 				throw any
 			} finally {
+				for (line in shReturnStdoutTrimmed("docker exec -t erp-train ls -1 /abas/erp/ | grep '.\\.FEHL'").split("\n")) {
```

Comment:
Why not simply `docker exec -t erp-train ls -1 /abas/erp/*.FEHL`? Then you would not need a pipe.